src/uipath/_events/_events.py

```diff
@@ -21,12 +21,13 @@ class EvalSetRunCreatedEvent(BaseModel):
     entrypoint: str
     eval_set_id: str
     no_of_evals: int
-    evaluators: List[Any]
+    # skip validation to avoid abstract class instantiation
+    evaluators: SkipValidation[List[AnyEvaluator]]
```

Comment:
`EvalSetRunCreatedEvent` should not be a `BaseModel` IMO. Please switch to `@dataclass` instead..